dashboard/app/assets/javascripts/application.js.erb

```diff
@@ -11,7 +11,7 @@
 //= require jquery
 //= require jquery_ujs
 //= require jquery-ui
-//= require video
+// (Former location of video.js; reference while converting to code-studio.js)
```

Comment:
Keeping this note so we can preserve the order of these includes as we move things over to code-studio.js.